examples/rocket_example/src/sqlx/mod.rs

```diff
@@ -0,0 +1,101 @@
+use rocket::fairing::{self, AdHoc};
+use rocket::response::status::Created;
+use rocket::serde::json::Json;
+use rocket::{Build, Rocket};
+use rocket_db_pools::{sqlx, Connection, Database};
+use sea_orm::entity::*;
+use sea_orm::RocketDbPool;
+
+mod setup;
+
+#[derive(Database, Debug)]
+#[database("rocket_example")]
+struct Db(RocketDbPool);
+
+type Result<T, E = rocket::response::Debug<sqlx::Error>> = std::result::Result<T, E>;
+
+mod post;
+pub use post::Entity as Post;
+
+#[post("/", data = "<post>")]
+async fn create(
+    conn: Connection<Db>,
+    post: Json<post::Model>,
+) -> Result<Created<Json<post::Model>>> {
+    let _post = post::ActiveModel {
+        title: Set(post.title.to_owned()),
+        text: Set(post.text.to_owned()),
+        ..Default::default()
+    }
+    .save(&conn)
+    .await
+    .expect("could not insert post");
+
+    Ok(Created::new("/").body(post))
+}
+
+#[get("/")]
+async fn list(conn: Connection<Db>) -> Result<Json<Vec<i32>>> {
+    let ids = Post::find()
+        .all(&conn)
+        .await
+        .expect("could not retrieve posts")
+        .into_iter()
+        .map(|record| record.id.unwrap())
+        .collect::<Vec<_>>();
+
+    Ok(Json(ids))
+}
+
+#[get("/<id>")]
+async fn read(conn: Connection<Db>, id: i64) -> Option<Json<post::Model>> {
+    let post: Option<post::Model> = Post::find_by_id(id)
+        .one(&conn)
+        .await
+        .expect("could not find post");
+
+    match post {
+        None => None,
+        Some(post) => Some(Json(post)),
+    }
+}
+
+#[delete("/<id>")]
+async fn delete(conn: Connection<Db>, id: i32) -> Result<Option<()>> {
+    let post: post::ActiveModel = Post::find_by_id(id)
+        .one(&conn)
+        .await
+        .unwrap()
+        .unwrap()
+        .into();
+    let result = post.delete(&conn).await.unwrap();
+
+    Ok((result.rows_affected == 1).then(|| ()))
+}
+
+#[delete("/")]
+async fn destroy(conn: Connection<Db>) -> Result<()> {
+    let _result = Post::delete_many().exec(&conn).await.unwrap();
+    Ok(())
+}
+
+async fn run_migrations(rocket: Rocket<Build>) -> fairing::Result {
+    #[cfg(feature = "sqlx-mysql")]
+    let db_url = "mysql://root:@localhost/rocket_example";
+    #[cfg(feature = "sqlx-postgres")]
+    let db_url = "postgres://root:root@localhost/rocket_example";
+
+    let conn = sea_orm::Database::connect(db_url).await.unwrap();
+
+    let _create_post_table = setup::create_post_table(&conn).await;
+    Ok(rocket)
+}
```

Comment:
Can we retrieve the db_url defined in `examples/rocket_example/Rocket.toml` from `Rocket<Build>`?

Reply:
I guess we can get the connection directly?
https://github.com/SergioBenitez/Rocket/blob/8cae077ba1d54b92cdef3e171a730b819d5eeb8e/examples/databases/src/sqlx.rs#L71-L82

Reply:
Yes, seems so